source/tutorials/minio-installation.rst

```diff
@@ -157,11 +177,34 @@ version of the ``minio`` server process:
 
 The command uses the following options:
 
-- ``-e MINIO_ACCESS_KEY`` and ``-e MINIO_SECRET_KEY`` for configuring the
-   :ref:`root <minio-users-root>` user credentials.
+.. list-table::
+   :widths: 40 60
+   :width: 100%
+
+   * - :envvar:`MINIO_ROOT_USER_FILE`
+     - The access key for the :ref:`root <minio-users-root>` user.
+
+       Replace this value with a unique, random, and long string. 
+
+   * - :envvar:`MINIO_ROOT_PASSWORD_FILE`
+     - The corresponding secret key to use for the 
+       :ref:`root <minio-users-root>` user.
+
+       Replace this value with a unique, random, and long string.
+
+   * - :envvar:`MINIO_KMS_SECRET_KEY`
+     - The key to use for encrypting the MinIO backend (users, groups,
+       policies, and server configuration).
+
+       Replace this value with a 256-bit base64-encrypted string:
```

Comment:
The format of `MINIO_KMS_SECRET_KEY` is `MINIO_KMS_SECRET_KEY=<key-name>:<base64-value>` (See: https://github.com/minio/minio/blob/master/docs/kms/IAM.md#minio-kms-quick-start )

Reply:
Fixed throughout.